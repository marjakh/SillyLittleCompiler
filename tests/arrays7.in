let a = new Array(10);
let i = 0;
while (i < 10) {
  a[i] = 65536 + i;
  i = i + 1;
}
i = 0;
while (i < 10) {
  print(a[i]);
  i = i + 1;
}
