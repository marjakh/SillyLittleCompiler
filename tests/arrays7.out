65536
65537
65538
65539
65540
65541
65542
65543
65544
65545
